Swift/ChartboostExample copy-Info.plist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe this file is not needed. There's already an Info.plist inside the project. Can we remove this?

Reply:
Yes, I'll switch the pods target to use the same Info.plist, thanks for that!

Reply:
Actually, I see now that this is the Info.plist required for the Pods target.
I think we should rename it to "ChartboostExample-Pods-Info.plist", same as the Obj-C one, put it in the same folder as the other Info.plist (I would rename that one to "ChartboostExample-Info.plist", same as the Obj-C one), and fix the link to it in the project's Build Settings (Packaging > Info.plist file).
Also make sure that both plist files have locations "relative to group" and not "absolute path)

Reply:
Nvm what I just said, sharing the same "Info.plist" makes a lot of sense